Dockerfile.ubuntu

```diff
@@ -54,8 +49,8 @@ WORKDIR $GF_PATHS_HOME
 
 COPY conf conf
 
-# We need font libs for phantomjs, and curl should be part of the image
-RUN apt-get update && apt-get upgrade -y && apt-get install -y ca-certificates libfontconfig1 curl
```

Comment:
If you run the image renderer plugin inside the docker container as a normal plugin won't that need some fonts?

Reply:
Probably, but that doesn't work anyway. However, we have instructions for how to [build a custom image with image renderer plugin support](https://grafana.com/docs/grafana/latest/installation/docker/#build-with-grafana-image-renderer-plugin-pre-installed). The idea/recommendation is to run remote render using docker since you already run Grafana in docker.

Reply:
Going to verify that the custom build still work without libfontconfig1